program/rust/src/c_oracle_header.rs

```diff
@@ -18,6 +18,9 @@ include!("../bindings.rs");
 pub trait PythAccount: Pod {
     const ACCOUNT_TYPE: u32;
     const INITIAL_SIZE: u32;
+    fn minimum_size() -> usize {
```

Comment:
document the difference between INITIAL_SIZE and minimum_size. this is going to get confusing